packages/earn-controller/src/EarnController.ts

```diff
@@ -242,8 +314,20 @@ export class EarnController extends BaseController<
       },
     });
 
-    this.#initializeSDK();
+    this.#env = env;
+
+    this.#earnApiService = new EarnApiService(this.#env);
+
+    // temporary array of supported chains
+    // TODO: remove this once we export a supported chains list from the sdk
+    // from sdk or api to get lending and pooled staking chains
+    this.#supportedPooledStakingChains = [1, 560048];
+
+    this.#addTransactionFn = addTransactionFn;
+
+    this.#initializeSDK().catch(console.error);
     this.refreshPooledStakingData().catch(console.error);
+    this.refreshLendingData().catch(console.error);
```

Comment:
Perhaps we ought to document in the changelog that lending data is now captured when the EarnController is initialized?